modules/large-deployments/pages/hub-auth.adoc

```diff
@@ -170,25 +170,27 @@ The map has two entries:
 import xmlrpclib
 
 
-HUB_URL = "http://localhost:8888/hub/rpc/api"
-HUB_LOGIN = "admin"
-HUB_PASSWORD = "admin"
 
-client = xmlrpclib.Server(<HUB_URL>, verbose=0)
+HUB_XMLRPC_API_URL = "<HUB_XMLRPC_API_URL>"
+HUB_USERNAME = "<USERNAME>"
+HUB_PASSWORD = "<PASSWORD>"
 
-hubKey = client.hub.loginWithAutoconnectMode(<HUB_LOGIN>, <HUB_PASSWORD>)
+client = xmlrpclib.Server(<HUB_XMLRPC_API_URL>, verbose=0)
```

Comment:
```suggestion
client = xmlrpclib.Server(HUB_XMLRPC_API_URL, verbose=0)
```

That seems incorrect syntax, I hope my correction is correct, please test all examples manually.